mixer/test/client/check_report/check_report_test.go

```diff
@@ -24,6 +24,7 @@ import (
 // Check attributes from a good GET request
 const checkAttributesOkGet = `
 {
+  "context.reporter.uid" : "*",
```

Comment:
Should those really be `"*"` (i.e. any) and not `""` (i.e. empty)?

Reply:
I'm not sure actually. @mandarjog I see you added these attributes in proxy, WDYT?

Reply:
Those are empty for now, so let's leave them like that for the time being, to avoid silent breakage in the future when someone adds real value, but doesn't update tests (since they would still match `"*"`), and later something breaks and the value won't be emitted anymore, but the tests will still pass, because empty value matches `"*"`.

I pulled your second commit into my original change in #8606, let's see if it works :)

Reply:
Sounds good.
Only the lint is failing in this PR, I think you might need to run `dep ensure` in your local repo to fix it.

Reply:
Yeah, I did that before pushing, so it should be all good. Thanks for the help!